ui/raidboss/emulator/data/NetworkLogConverter.ts

```diff
@@ -0,0 +1,41 @@
+import EventBus from '../EventBus';
+import LineEvent from './network_log_converter/LineEvent';
+import LogRepository from './network_log_converter/LogRepository';
+import ParseLine from './network_log_converter/ParseLine';
+
+export default class NetworkLogConverter extends EventBus {
+  convertFile(data: string): LineEvent[] {
+    const repo = new LogRepository();
+    return this.convertLines(
+        // Split data into an array of separate lines, removing any blank lines.
+        data.split(NetworkLogConverter.lineSplitRegex).filter((l) => l !== ''),
+        repo,
+    );
+  }
+
+  convertLines(lines: (string | LineEvent)[], repo: LogRepository): LineEvent[] {
+    repo = repo || new LogRepository();
```

Comment:
Is this necessary? It looks like from both the type and the callers that repo always exists.

Reply:
This is leftover code from the original bootstrapping raidemulator test framework I was using to compare network lines to converted non-network lines pulled from ACT. Removed for next commit.